src/pages/fresh_install/FreshInstallPage.css

```diff
@@ -0,0 +1,100 @@
+.DivButtons {
+    margin-top: 3vh;
+}
+
+.ButtonCelsius {
+    margin-right: 2rem;
+}
+
+.Button {
+    width: 120px;
+    height: 45px;
+    object-fit: contain;
+    border-radius: 15px;
+    box-shadow: 0 5px 10px 0 rgba(0, 0, 0, 0.15);
+    background-color: rgba(255, 255, 255, 0.9);
+    display: flex;
+    align-items: center;
+}
+
+.ButtonActive {
+    width: 120px;
+    height: 45px;
+    object-fit: contain;
+    border-radius: 15px;
+    box-shadow: 0 5px 10px 0 rgba(0, 0, 0, 0.15);
+    background-color: rgba(255, 255, 255, 0.35);
+    display: flex;
+    align-items: center;
+}
+
+.DivContent {
+    text-align: center;
+    display: flex;
+    flex-direction: row;
+    justify-content: center;
+}
+
+.Next {
+    display: flex;
+    align-items: center;
+    height: 54px;
+    color: #ffffff;
+}
+
+.DivNext {
+    padding-top: 26vh;
+}
+
+.Description {
+    display: flex;
+    align-items: center;
+    height: 54px;
+    color: #ffffff;
+}
+
+.DivDescription {
+    padding-top: 40vh;
+}
+
+.Text {
+    font-family: Avenir;
+    font-size: 20px;
+    font-weight: 500;
+    font-style: normal;
+    font-stretch: normal;
+    line-height: normal;
+    letter-spacing: normal;
+    text-align: center;
+}
+
+.ButtonText {
+    width: 120px;
+    height: 25px;
+    color: #2f72bb;
+    display: inline-block;
+    vertical-align: middle;
+}
+
+.TextActive {
+    color: white;
+}
+
+.FreshInstallPage {
+    min-height: 100vh;
+    max-width: 100%;
+    background-image: url("../../images/bg-settings-first-access@2x.png");
+    background-size: cover;
+}
+
+.Path4 {
```

Comment:
What does `Path4` mean? Can we use another more meaningful name?

Reply:
Path4 is the name of the file that represents the "arrow" near to "Next" text on the bottom of the screen. I'll rename it. Thank you.